plugins/GSdx/GSDrawScanlineCodeGenerator.x86.cpp

```diff
@@ -2889,7 +2889,7 @@ void GSDrawScanlineCodeGenerator::ReadTexel_SSE(int pixels, int mip_offset)
 				movdqa(ptr[&m_local.temp.test], xmm7);
 			}
 
-			for(int j = 0; j < 4; j++)
+			for(uint8 j = 0; j < 4; j++)
```

Comment:
I have seen cases where even a `for ( i = 0; i < 1; i++ )` loop wouldn't be removed in Release mode though... so it's probably best to ensure it's the best code possible first.

Or just unroll it manually and get it over with.